internal/mysql/schema.go

```diff
@@ -63,6 +63,18 @@ func (s *Schema) Add(ddl *sqlparser.DDL) {
 			panic(fmt.Sprintf("failed to parse table \"%s\" schema.", name))
 		}
 		s.tables[name] = ddl.TableSpec.Columns
+	case "rename":
+		if len(ddl.FromTables) != 1 || len(ddl.ToTables) != 1 {
+			panic(fmt.Errorf("rename without one 'from' table and one 'to' table: %#v", ddl))
```

Comment:
Will switch to returning an error if https://github.com/kyleconroy/sqlc/pull/608 gets merged